Code/MinimalLib/cffiwrapper.h

```diff
@@ -122,13 +122,15 @@ RDKIT_RDKITCFFI_EXPORT short fragment_parent(char **pkl, size_t *pkl_sz,
 
 // coordinates
 RDKIT_RDKITCFFI_EXPORT void prefer_coordgen(short val);
+RDKIT_RDKITCFFI_EXPORT short has_2d_coords(char *mol_pkl, size_t mol_pkl_sz);
```

Comment:
The actual implementation of this checks whether or not the molecule has a conformer, not 2D coords.
How about renaming this to `has_conformer`?

Reply:
Good point, I copy pasted the `set_2d_coords` signature and left `2d` in by mistake. I renamed to `has_coords` for consistency with the `set_*` functions.